themes/default/content/docs/pulumi-cloud/oidc/client/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title_tag: OpenID Connect Client Integration for Pulumi
+meta_desc: This page provides an overview of how to configure OpenID Connect integration between
+           third party OIDC Providers and Pulumi.
+title: OpenID Client
+h1: OpenID Connect integration
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    pulumicloud:
+        parent: openid-connect
+        identifier: openid-connect-client
+        weight: 4
+---
+
+Pulumi supports establishing trust relationships with third party OIDC providers by leveraging id_tokens and allowing it to be exchanged by a short-lived Pulumi access token. This mechanism enhances security by eliminating the necessity for hardcoded credentials.
```

Comment:
```suggestion
Pulumi supports establishing trust relationships with third party OIDC providers by leveraging id_tokens and allowing it to be exchanged for a short-lived Pulumi access token. This mechanism enhances security by eliminating the necessity for hardcoded credentials.
```

Reply:
I have seen this concept of avoiding the need for "secret zero" in docs across industry. Should we allude to that for SEO purposes?

Reply:
I will add something about that

Reply:
got into the buzz word train and added zero trust to the mix too, let me know what do you think